src/controls/filePicker/MultipleUploadFilePicketTab/MultipleUploadFilePicketTab.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+
+import { IMultipleUploadFilePicketTabProps } from './IMultipleUploadFilePicketTabProps';
+import { IMultipleUploadFilePickerTabState } from './IMultipleUploadFilePicketTabState';
+import { IFilePickerResult } from '../FilePicker.types';
+import { GeneralHelper } from '../../../common/utilities/GeneralHelper';
+import { PrimaryButton, DefaultButton } from 'office-ui-fabric-react/lib/components/Button';
+import { css } from 'office-ui-fabric-react/lib/Utilities';
+import { DragDropFiles } from '../../dragDropFiles/DragDropFiles';
+
+import * as strings from 'ControlStrings';
+import styles from './MultipleUploadFilePicketTab.module.scss';
+
+
+
+export default class MultipleUploadFilePicketTab extends React.Component<IMultipleUploadFilePicketTabProps, IMultipleUploadFilePickerTabState> {
+  constructor(props: IMultipleUploadFilePicketTabProps) {
+    super(props);
+    this.state = {
+      filePickerResult: undefined
+    };
+  }
+
+  private displayFileNames = (filesResult) => {
+    const result = [];
+    for (var i = 0; i < filesResult.length; i++) {
+      result.push(<div className={styles.localTabFilename}>{filesResult[i].name}</div>);
```

Comment:
This loop is generating a warning in the browser console as the repeated elements don't have a key property
```suggestion
      result.push(<div key={i.toString()} className={styles.localTabFilename}>{filesResult[i].name}</div>);
```

Reply:
@aaclage I have just added the `key` property to the version in dev-v2 branch

Reply:
@joelfmrodrigues already making the changes.